packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -272,36 +265,30 @@ async function setupServices(
   // Cheesy hack for easy use of manual price authority
   const pa = Array.isArray(priceOrList)
     ? makeScriptedPriceAuthority({
-        actualBrandIn: aethBrand,
-        actualBrandOut: runBrand,
-        priceList: priceOrList,
-        timer,
-        quoteMint,
-        unitAmountIn,
-        quoteInterval,
-      })
+      actualBrandIn: aethBrand,
```

Comment:
needs fixing. you can avoid CI errors of formatting with [format-on-save](https://github.com/Agoric/agoric-sdk/blob/b586fe8c13d9d180ddcc436116f46dcabe6b351b/scripts/configure-vscode.sh#L22-L23)

Reply:
I have format on save. No idea how this was incorrectly indented in the first place.